src/backend/routes/aliases.ts

```diff
@@ -46,9 +46,9 @@ router.get('*', verifyToken, async (req: Request, res: Response) => {
       }
     }
   } catch (err) {
-    res.status(400).json({
-      success: false,
-      error: err,
+    res.status(404).render('error', {
```

Comment:
Looks like all the errors thrown in the code above will be handed as 404, including internal errors and others. Maybe we should check for the 'Alias not found' error? 

And it would be better to use [Custom Errors](https://javascript.info/custom-errors) instead of determining error type by text message (like 'Alias not found')

Reply:
Done